src/Innertube.ts

```diff
@@ -131,6 +134,30 @@ export default class Innertube {
     return new VideoInfo([ response ], this.actions, cpn);
   }
 
+  /**
+   * Retrieves shorts info.
+   * @param short_id - The short id.
+   * @param client - The client to use.
+   */
+  async getShortsWatchItem(short_id: string, client?: InnerTubeClient): Promise<ShortsVideoInfo> {
+    throwIfMissing({ short_id });
+
+    const response = await this.actions.execute(
+      Reel.WatchEndpoint.PATH, Reel.WatchEndpoint.build({
+        short_id: short_id,
+        client: client
+      })
+    );
+
+    const sequenceResponse = await this.actions.execute(
+      Reel.WatchSequenceEndpoint.PATH, Reel.WatchSequenceEndpoint.build({
+        sequenceParams: encodeReelSequence(short_id)
+      })
+    );
```

Comment:
Please use `Promise.all` instead of awaiting them individually, that will allow the requests to run in parallel, you can use the `getInfo` function for inspiration if you want.

Reply:
Done 🙂